diff -p1 -Naur Paket-1.5.0.old/Paket.sln Paket-1.5.0/Paket.sln
--- Paket-1.5.0.old/Paket.sln	2015-05-21 17:14:09.000000000 +0100
+++ Paket-1.5.0/Paket.sln	2015-05-25 01:45:51.662629602 +0100
@@ -46,4 +46,2 @@ Project("{2150E333-8FDC-42A3-9474-1A3956
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Paket.Tests", "tests\Paket.Tests\Paket.Tests.fsproj", "{E789C72A-5CFD-436B-8EF1-61AA2852A89F}"
-EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Paket", "src\Paket\Paket.fsproj", "{09B32F18-0C20-4489-8C83-5106D5C04C93}"
@@ -76,6 +74,2 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{E789C72A-5CFD-436B-8EF1-61AA2852A89F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{E789C72A-5CFD-436B-8EF1-61AA2852A89F}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{E789C72A-5CFD-436B-8EF1-61AA2852A89F}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{E789C72A-5CFD-436B-8EF1-61AA2852A89F}.Release|Any CPU.Build.0 = Release|Any CPU
 		{09B32F18-0C20-4489-8C83-5106D5C04C93}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
diff -p1 -Naur Paket-1.5.0.old/src/Paket.Core/Paket.Core.fsproj Paket-1.5.0/src/Paket.Core/Paket.Core.fsproj
--- Paket-1.5.0.old/src/Paket.Core/Paket.Core.fsproj	2015-05-21 17:14:09.000000000 +0100
+++ Paket-1.5.0/src/Paket.Core/Paket.Core.fsproj	2015-05-25 01:45:09.167927089 +0100
@@ -69,10 +69,4 @@
   <ItemGroup>
-    <Compile Include="..\..\paket-files\fsprojects\Chessie\src\Chessie\ErrorHandling.fs">
-      <Paket>True</Paket>
-      <Link>ErrorHandling.fs</Link>
-    </Compile>
-    <Compile Include="..\..\paket-files\fsharp\FAKE\src\app\FakeLib\Globbing\Globbing.fs">
-      <Paket>True</Paket>
-      <Link>Globbing.fs</Link>
-    </Compile>
+    <Compile Include="ErrorHandling.fs" />
+    <Compile Include="Globbing.fs" />
     <Compile Include="Async.fs" />
diff -p1 -Naur Paket-1.5.0.old/tests/Paket.Tests/Paket.Tests.fsproj Paket-1.5.0/tests/Paket.Tests/Paket.Tests.fsproj
--- Paket-1.5.0.old/tests/Paket.Tests/Paket.Tests.fsproj	2015-05-21 17:14:09.000000000 +0100
+++ Paket-1.5.0/tests/Paket.Tests/Paket.Tests.fsproj	2015-05-25 01:45:37.959047965 +0100
@@ -67,9 +67,4 @@
   <ItemGroup>
-    <Compile Include="FsCheckAddin.fs">
-      <Paket>True</Paket>
-    </Compile>
-    <Compile Include="..\..\paket-files\forki\FsUnit\FsUnit.fs">
-      <Paket>True</Paket>
-      <Link>FsUnit.fs</Link>
-    </Compile>
+    <Compile Include="FsCheckAddin.fs" />
+    <Compile Include="FsUnit.fs" />
     <Compile Include="AssemblyInfo.fs" />
